client/connect-account-page/index.tsx

```diff
@@ -227,35 +316,64 @@ const ConnectAccountPage: React.FC = () => {
 								disabled={ isSubmitted }
 								onClick={ handleSetup }
 							>
-								{ wcpaySettings.isJetpackConnected
-									? strings.button.jetpack_connected
-									: strings.button.jetpack_not_connected }
+								{ ctaLabel }
 							</Button>
+							{
+								// Only show the reset button if an account is connected and didn't complete KYC, or if we are in dev mode.
+								isAccountConnected &&
+									( ! wcpaySettings.accountStatus
+										.detailsSubmitted ||
+										isInDevMode() ) && (
+										<Button
+											variant={ 'tertiary' }
+											onClick={ () =>
+												setModalVisible( true )
+											}
+										>
+											{ strings.button.reset }
+										</Button>
+									)
+							}
 						</div>
 					</Card>
-					{ incentive && <Incentive { ...incentive } /> }
-					<Panel className="connect-account-page__sandbox-mode-panel">
-						<PanelBody
-							title={ strings.sandboxMode.title }
-							initialOpen={ false }
-						>
-							<InlineNotice
-								icon
-								status="info"
-								isDismissible={ false }
-							>
-								{ strings.sandboxMode.description }
-							</InlineNotice>
-							<Button
-								variant="secondary"
-								isBusy={ isSandboxModeClicked }
-								disabled={ isSandboxModeClicked }
-								onClick={ handleEnableSandboxMode }
-							>
-								{ strings.button.sandbox }
-							</Button>
-						</PanelBody>
-					</Panel>
+					{
+						// Only show the incentive if an account is NOT connected.
+						! isAccountConnected && incentive && (
+							<Incentive { ...incentive } />
+						)
+					}
+					{
+						// Only show the sandbox mode panel if an account is NOT connected.
+						! isAccountConnected && (
+							<Panel className="connect-account-page__sandbox-mode-panel">
```

Comment:
Do we have this as a component? If yes let's use it

Reply:
is it `TestModeNotice`?

Reply:
I don't follow: that is the builder flow section of the Connect page.